src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.Helpers/HelperUtils.cs

```diff
@@ -213,5 +213,52 @@ string id
             return policyName;
         }
     
+        public static string GetSubscriptionIdFromId(Dictionary<CmdletModel.UriEnums, string> keyValuePairs, string id)
+        {
+            string subscriptionId = string.Empty;
+
+            if (keyValuePairs.ContainsKey(CmdletModel.UriEnums.Subscriptions))
+            {
+                subscriptionId = keyValuePairs[CmdletModel.UriEnums.Subscriptions];
+            }
+            else
+            {
+                throw new ArgumentException(string.Format("", CmdletModel.UriEnums.Subscriptions.ToString(), id));
+            }
+
+            return subscriptionId;
+        }
+
+        public static string GetResourceGroupNameFromId(Dictionary<CmdletModel.UriEnums, string> keyValuePairs, string id)
+        {
+            string resourceGroupName = string.Empty;
+
+            if (keyValuePairs.ContainsKey(CmdletModel.UriEnums.ResourceGroups))
+            {
+                resourceGroupName = keyValuePairs[CmdletModel.UriEnums.ResourceGroups];
+            }
+            else
+            {
+                throw new ArgumentException(string.Format("", CmdletModel.UriEnums.ResourceGroups.ToString(), id));
```

Comment:
You have provided no format.  Nothing is going to be printed in the ArgumentException